packages/vue-components/src/__tests__/Questions.spec.js

```diff
@@ -274,6 +274,213 @@ describe('Checkbox Questions and QOptions', () => {
   });
 });
 
+describe('Blank Questions and QOptions', () => {
+  test('of unanswered with shown hint and header renders correctly', async () => {
+    const option = {
+      render(h) {
+        return h(QOption, { props: { keywords: 'key' } });
+      },
+    };
+
+    const wrapper = mount(Question, {
+      propsData: { type: 'blanks' },
+      slots: {
+        default: [
+          'Question content',
+          option,
+          option,
+          option,
+          option,
+        ],
+        header: 'unanswered blanks question header',
+        hint: 'blanks question hint',
+      },
+      provide: DEFAULT_INJECTIONS,
+      stubs: DEFAULT_STUBS,
+    });
+
+    // Click hint
```

Comment:
@weiquu Sorry for the late review, just a minor nit on maintaining consistency if possible.
In your comments within this file and the next, you have:
- `// Click hint`,
- `// click 'check'`,
- `// click 'hint'`,
- `// Click check of blanks question`

Sometimes there's a quotation, sometimes the comment is capitalized, and sometimes not. Perhaps can standardize a little in future PRs.